### ISSUE TITLE
NTR: Creatinine low in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8810]
New term request Creatinine low in urine (novotn_041028230949): 24 hour excretion  UMLS: 1096344

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000115
- label: Decreased urine creatinine level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: 
- def: The amount of creatinine in the urine, normalized for urine concentration, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8810
- synonyms:
